tests/rfc003_ether_htlc.rs

```diff
@@ -224,4 +233,56 @@ fn given_short_zero_secret_htlc_should_not_redeem() {
         client.eth_balance_of(htlc),
         U256::from("0400000000000000000")
     );
+    assert_return_data(&client, transaction_receipt, INVALID_SECRET);
+}
+
+#[test]
+fn given_invalid_secret_htlc_should_revert_tx_with_error() {
```

Comment:
No test before? same 😱

Reply:
Same as above, there was a quick check in another test. 
I removed that and create an explicit test. 

Look for this in this PR: 
```
let transaction_receipt = client.send_data(htlc_address, Some(Bytes(b"I'm a h4x0r".to_vec())));	
assert_that(&transaction_receipt.logs).has_length(0);
```